bundles/org.openhab.binding.nikobus/README.md

```diff
@@ -33,6 +33,12 @@ The bridge enables communication with other Nikobus components:
 - `rollershutter-module` - Nikobus roller shutter module,
 - `push-button` - Nikobus physical push button.
 
+## Warning : PC-Link software addresses writing convention 
```

Comment:
Rather than a big chapter for a warning, I would rather imagine to include in the "Modules" chapter when discussing about module address.

Reply:
Ok, it is not so dramatic to have this like that.